pkg/operator/defaultscccontroller/resourcehandler.go

```diff
@@ -0,0 +1,61 @@
+package defaultscccontroller
+
+import (
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog"
+)
+
+// NewResourceHandler returns a cache.ResourceEventHandler appropriate for
+// reconciliation of SecurityContextConstraints object(s).
+func NewResourceHandler(queue workqueue.RateLimitingInterface) ResourceHandler {
```

Comment:
does this really need this61 line file instead of an inline event handler in the main controller?

Reply:
are you referring to this?
```
func defaultSCCControllerEventHandler(queue workqueue.RateLimitingInterface) cache.ResourceEventHandler {
	addFunc := func(key string, queue workqueue.RateLimitingInterface) {
		namespace, name, err := cache.SplitMetaNamespaceKey(key)
		if err != nil {
			return
		}

		queue.Add(types.NamespacedName{
			Namespace: namespace,
			Name:      name,
		})
	}

	return cache.ResourceEventHandlerFuncs{
		AddFunc: func(obj interface{}) {
			key, err := cache.MetaNamespaceKeyFunc(obj)
			if err != nil {
				klog.Errorf("[%s] OnAdd: could not extract key, type=%T- %s", ControllerName, obj, err.Error())
				return
			}

			addFunc(key, queue)
		},

		UpdateFunc: func(old, new interface{}) {
			key, err := cache.MetaNamespaceKeyFunc(new)
			if err != nil {
				klog.Errorf("[%s] OnUpdate: could not extract key, type=%T- %s", ControllerName, new, err.Error())
				return
			}

			addFunc(key, queue)
		},
		DeleteFunc: func(obj interface{}) {
			key, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
			if err != nil {
				klog.Errorf("[%s] OnDelete: could not extract key, type=%T - %s", ControllerName, obj, err.Error())
				return
			}

			addFunc(key, queue)
		},
	}
}
```
or are you referring to an existing resource handler that I can use off the shelf?

Reply:
yes, this. ERROVERABSTRACTION.